Reuters.js

```diff
@@ -164,56 +184,55 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
+				"date": "Mon Nov 14 21:16:28 UTC 2011",
```

Comment:
I'm worried how Zotero is going to parse that date. We should normalize that or remove the timestamp